modules/services/workspace-service/src/main/java/org/ballerinalang/composer/service/workspace/app/StartBrowser.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ballerinalang.composer.service.workspace.app;
+
+import java.io.IOException;
+import java.util.Locale;
+
+/**
+ * Starting the default browser once the composer starts
+ */
+public class StartBrowser {
+
+    public static void startInDefaultBrowser(String url) throws IOException {
+        Locale locale = Locale.ENGLISH;
+        String os = System.getProperty("os.name").toLowerCase(locale);
+        Runtime rt = Runtime.getRuntime();
+            if (os.indexOf("win") >= 0) {
```

Comment:
you can use LaunchUtils.isWindows method to check this https://github.com/ballerinalang/composer/blob/master/modules/services/workspace-service/src/main/java/org/ballerinalang/composer/service/workspace/launcher/util/LaunchUtils.java#L32

Reply:
Fixed with fbbb851